test/fixtures/b.js

```diff
@@ -1,3 +1,6 @@
 module.exports = function b() {
 	return "This is b";
-};
\ No newline at end of file
+};
+
+// Test CJS top-level return
+return;
```

Comment:
Can we add new test instead modify current?

Reply:
Thanks for reviewing. The idea is for this to be exactly the same change as https://github.com/webpack/webpack/pull/8510 , which was already merged into 'next'. If you really prefer the test to be split it up I can to that, but it seems to me that that does not benefit understandability when comparing the branches.

Reply:
:+1: